### ISSUE TITLE
Changelog in releases

### ISSUE THREAD

**Issue by NoxInmortus** [wekan/wekan#3017]
Hello,

I would like to ask for the changelog to be added/deported to the releases as it can then be used for RSS news.

If it wasnt known : https://github.com/wekan/wekan/releases.atom provides an rss compatible flux that can be read easily with any RSS agregator supporting this format.

**Comment by xet7** [comment]
I added RSS feed to Wekan webpage https://wekan.github.io and for that I needed to add content to those release tags.

I added release info for Wekan v3.88 - v3.97. Because about 637 are watching this repo, all those got 9 emails about Wekan releases to their mailbox.

That atom feed only shows newest 9 releases.

Well, I was thinking about using script and GitHub API to add releases to v1.08 - v3.87 but that would mean 297 emails to every one of those 637 users, but I think that would be too much. I already got @FanyangMengDev asking about Wekan release email flood from those 9 emails.

So I think this is enough.
